tests/test_elements.py

```diff
@@ -0,0 +1,1878 @@
+import os, sys
+
+THIS_FOLDER = os.path.dirname(os.path.abspath(__file__)) + '/..'
+sys.path.insert(1, THIS_FOLDER)
+
+from vyxal.transpile import *
+from vyxal.context import Context
+from vyxal.elements import *
+from vyxal.LazyList import *
+def test_LogicalNot():
+	stack = [1]; expected = 0
+	ctx = Context()
+	code = transpile('¬')
+	exec(code)
+	assert simplify(stack[-1]) == expected
+
+	stack = [0]; expected = 1
+	ctx = Context()
+	code = transpile('¬')
+	exec(code)
+	assert simplify(stack[-1]) == expected
+
+	stack = ["abc"]; expected = 0
+	ctx = Context()
+	code = transpile('¬')
+	exec(code)
+	assert simplify(stack[-1]) == expected
+
+	stack = [""]; expected = 1
+	ctx = Context()
+	code = transpile('¬')
+	exec(code)
+	assert simplify(stack[-1]) == expected
+
+	stack = [[1,2,3]]; expected = 0
+	ctx = Context()
+	code = transpile('¬')
+	exec(code)
+	assert simplify(stack[-1]) == expected
+
+	stack = [[]]; expected = 1
+	ctx = Context()
+	code = transpile('¬')
+	exec(code)
+	assert simplify(stack[-1]) == expected
+
```

Comment:
Why three lines between functions? Do test functions have special rules in black, or has this not been run through a formatter yet?

Reply:
This hasn't been run through black, because it's automatically generated by the YAML.